packages/@ember/-internals/routing/lib/system/router.ts

```diff
@@ -467,8 +493,8 @@ class EmberRouter extends EmberObject {
       'application',
       { path: '/', resetNamespace: true, overrideNameAssertion: true },
       function(this: EmberRouterDSL) {
-        for (let i = 0; i < dslCallbacks.length; i++) {
-          dslCallbacks[i].call(this);
+        for (let dslCallback of dslCallbacks) {
```

Comment:
We cannot use `for...of` loops due to global `Symbol` requirement (`Symbol` is not present on IE11).

Reply:
gnrgs I hate this world. Sorry.